2996_reflection/reflection.md

```diff
@@ -4414,6 +4421,7 @@ consteval void g(std::meta::info r, X<false> xv) {
   - [#.#.#]{.pnum} Otherwise, if the `$template-name$` denotes a primary class template, primary variable template, or alias template, `$R$` represents that template.
 - [#.#]{.pnum} Otherwise, if the `$identifier$` names a type alias that was introduced by the declaration of a template parameter, `$R$` represents the underlying entity of that type alias. For any other `$identifier$` that names a type alias, `$R$` represents that type alias.
 - [#.#]{.pnum} Otherwise, if the `$identifier$` is a `$class-name$` or an `$enum-name$`, `$R$` represents the denoted type.
+- [#.#]{.pnum} Otherwise, if the `$identifier$` names a class member of an anonymous union ([class.union.anon]), `$R$` represents that class member.
```

Comment:
Do you need the word "class" here?

Reply:
I like it, Hubert LGTM'd it. Let's keep it for now; there is precedent elsewhere.